R/trim_amount.R

```diff
@@ -0,0 +1,18 @@
+#' Amount of Trimming
+#'
+#' Used in `recipes::step_impute_mean()`.
+#'
+#' @inheritParams Laplace
+#' @examples
+#' trim_amount()
+#' @export
+trim_amount <- function(range = c(0, 0.5), trans = NULL) {
+  new_quant_param(
+    type = "double",
+    range = range,
+    inclusive = c(TRUE, TRUE),
+    trans = trans,
+    label = c(trim_amount = "Kernel trim_amount"),
```

Comment:
It's not immediately obvious to me what this has to do with kernels. Is that intentional or some c+p left over? The labels generally try to explain what the parameter is, without repeating the name. Could you come up with a label that tries to include these two aspects?

Reply:
copy paste error....